Fixes Internal #4686

matthewhelmke opened the pull request:
## Type of change
This moves several pages that document chainctl features from varied locations in the docs into the chainctl Usage section. It fixes https://github.com/chainguard-dev/internal/issues/4686

### What are the acceptance criteria? 
Since this PR rearranges pages, I would really like @smythp to take a look and determine whether we need any rewrites to cover the moves.

I would also like @erikaheidi and @SharpRake  to take a look to see if they think the new titles and new page locations/order make sense. Note that the pages with short names and no verbs in the titles are slated to be expanded, so I'll rename them one by one as that happens in future work, but that's why there are two different title styles. The "Verb Something" titles are either existing or newer tutorial-ish pages and not solely reference like the others.

matthewhelmke committed:
Fixes Internal #4686

SharpRake reviewed (commented):
I think these changes are sound. I guess I'm a little hung up on moving `comparing-images.md` to the chainctl usage bucket. On one hand it makes sense since it's all about the chainctl images diff command, but on the other it feels closely really closely tied to Containers and to me it fits well in the "How to Use" bucket under Containers. 

I also left some thoughts around linktitles, let me know if that needs clarification

erikaheidi commented:
Something that stands out to me is the difference in the menu names, I really like the verb + something style from the first docs showing in the submenu, than it looks weird when mixed with "chainctl IAM", "chainctl events" etc. I see you mentioning these will also change in the future, so at least I would move the "Compare images with chainctl" a bit to the top to stay together with the other verb + something style (just grouping together the different styles basically).

I think Mark had a good point about consolidating some of the images-specific command references (such as "chainctl images history") in a subsection of its own, like "Working with Images". We could then in the future include other products as sub-folders too. But we risk duplicating what's in the Reference Docs, even if just for titles. Currently we have, for instance, two pages named "chainctl images":

- https://deploy-preview-2227--ornate-narwhal-088216.netlify.app/chainguard/chainctl/chainctl-docs/chainctl_images/
- https://deploy-preview-2227--ornate-narwhal-088216.netlify.app/chainguard/chainctl-usage/chainctl-images/

We may be more strategic on these title names to avoid the duplication.

matthewhelmke commented on content/chainguard/chainctl-usage/comparing-images.md:
```diff
@@ -1,6 +1,6 @@
 ---
 title: "How To Compare Chainguard Containers with chainctl"
-linktitle: "Compare with chainctl"
+linktitle: "Compare Images with chainctl"
```
I just got another idea. I think I'll try moving that page up so all the verb-titles are grouped and the other titles follow, then move those pages up into the verbed title order as they are expanded. That requires very little of the reader since they can see at a glance there is something different.

matthewhelmke commented:
> I think these changes are sound. I guess I'm a little hung up on moving `comparing-images.md` to the chainctl usage bucket. On one hand it makes sense since it's all about the chainctl images diff command, but on the other it feels closely really closely tied to Containers and to me it fits well in the "How to Use" bucket under Containers.
> 

I've been waffling about this move for exactly this reason. At the moment, the desire to have all the chainctl stuff in one place is winning out, but I bet we can find a suitable place in the Containers area to put a link to it.

matthewhelmke committed:
edits after review

smythp reviewed (approved):
As far as I can tell, you haven't moved, renamed, or deleted any files, so redirects won't be needed. Changing the weight shouldn't matter.

Erika reviewed from a semantic perspective so I won't weigh in on that unless you'd like another opinion.

matthewhelmke commented:
> As far as I can tell, you haven't moved, renamed, or deleted any files, so redirects won't be needed. Changing the weight shouldn't matter.
> 

I have moved three files:

- comparing-images.md
- how-to-install-chainctl.md
- manage-chainctl-config.md
